docker-compose.yml

```diff
@@ -23,7 +23,7 @@ services:
     ports:
       - "${FRONTEND_HOST_PORT}:80"  # Host Port:Container Port
     volumes:
-      - ./data:/app/data/csv:z  # Shared volume for CSV files
+      - ./data/csv:/app/data/csv:z  # Shared volume for CSV files
     networks:
       - broadcast-network
```

Comment:
the name is flare now so broadcast network does not make sense as the network name